internal/v1/handler_test.go

```diff
@@ -995,7 +995,6 @@ func TestGetCustomizations(t *testing.T) {
 				if result.Openscap == nil {
 					require.Nil(t, customizations.Openscap)
 				} else {
-					customizations.Openscap.ProfileId = string(profile)
```

Comment:
Intended change?

Reply:
Yea, I'm not sure why this was done in this way. Like it was setting the result before comparing it with the expected outcome… It was just a mock issue, somewhere the profile id's of the mock didn't match